daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/Daml3ScriptTestRunnerDev.scala

```diff
@@ -19,21 +19,27 @@ class Daml3ScriptTestRunnerDev extends DamlScriptTestRunner {
 
   override lazy val darFiles = List(daml3DarPath)
 
+  val expectedContractNotActiveResponse =
+    if (cantonFixtureDebugMode)
+      """FAILURE (com.daml.lf.engine.script.Runner$InterpretationError: Error: Unhandled Daml exception: DA.Exception.GeneralError:GeneralError@XXXXXXXX{ message = "contractNotActive no additional info" })"""
+    else
+      "SUCCESS"
```

Comment:
I dont understand the need for this condition, nor - now that i think about it, why we even have debug mode on.

Reply:
Yeah the guard wasn't necessary at all - still odd that we have `cantonFixtureDebugMode` on.

Reply:
It shouldnt be needed